pkg/manifest/os.go

```diff
@@ -822,6 +826,25 @@ func (p *OS) serialize() osbuild.Pipeline {
 		}
 	}
 
+	if len(p.CACerts) > 0 {
+		for _, cc := range p.CACerts {
+			certs, err := cert.ParseCerts(cc)
+			if err != nil {
+				panic(fmt.Errorf("failed to parse CA certificates: %v", err))
+			}
+
+			for _, c := range certs {
+				path := filepath.Join("/etc/pki/ca-trust/source/anchors", filepath.Base(c.SerialNumber.Text(16))+".pem")
+				f, err := fsnode.NewFile(path, nil, "root", "root", pem.EncodeToMemory(&pem.Block{Type: "CERTIFICATE", Bytes: c.Raw}))
+				if err != nil {
+					panic(err)
+				}
+				pipeline.AddStages(osbuild.GenFileNodesStages([]*fsnode.File{f})...)
+			}
+		}
+		pipeline.AddStage(osbuild.NewCAStageStage())
+	}
+
```

Comment:
Please move this whole block up before the selinux stage is added (L798), so that any newly created files get labelled correctly.